src/db/migration/V1_1727687613772__Data_export_views.sql

```diff
@@ -0,0 +1,468 @@
+        EXECUTE format('ALTER TABLE %I.palaute_viestit ADD CONSTRAINT palautteet_fkey FOREIGN KEY (palaute_id) REFERENCES %I.palautteet (id)', target_schema, target_schema);
+        EXECUTE format('ALTER TABLE %I.palaute_tapahtumat ADD CONSTRAINT palautteet_fkey FOREIGN KEY (palaute_id) REFERENCES %I.palautteet (id)', target_schema, target_schema);
+
+        RETURN true;
+    END IF;
+END;
+$BODY$
+    LANGUAGE plpgsql;
+
+
+CREATE OR REPLACE FUNCTION refresh_reporting (target_schema TEXT) RETURNS BOOLEAN AS
+$BODY$
+DECLARE tbl RECORD;
+BEGIN
+    IF lower(target_schema) = 'public' THEN
+        RETURN false;
+    ELSE
+        EXECUTE format('DROP SCHEMA IF EXISTS %I CASCADE', target_schema);
+        EXECUTE format('CREATE SCHEMA %I', target_schema);
+        FOR tbl IN
+            SELECT *
+            FROM information_schema.tables
+            WHERE table_schema = 'public'
+              AND table_type = 'BASE TABLE'
+              AND table_name NOT IN ('flyway_schema_history',
+                                     'aiemmin_hankitun_ammat_tutkinnon_osan_naytto',
+                                     'aiemmin_hankitut_ammat_tutkinnon_osat',
+                                     'aiemmin_hankitun_paikallisen_tutkinnon_osan_arvioijat',
+                                     'aiemmin_hankitun_paikallisen_tutkinnon_osan_naytto',
+                                     'aiemmin_hankitut_paikalliset_tutkinnon_osat',
+                                     'aiemmin_hankitun_yhteisen_tutkinnon_osan_arvioijat',
+                                     'aiemmin_hankitun_yhteisen_tutkinnon_osan_naytto',
+                                     'aiemmin_hankitut_yhteiset_tutkinnon_osat',
+                                     'aiemmin_hankitun_yto_osa_alueen_naytto',
+                                     'aiemmin_hankitut_yto_osa_alueet',
+                                     'hankittavan_paikallisen_tutkinnon_osan_naytto',
+                                     'hankittavan_paikallisen_tutkinnon_osan_osaamisen_hankkimistavat',
+                                     'hankittavat_paikalliset_tutkinnon_osat',
+                                     'hankittavan_ammat_tutkinnon_osan_naytto',
+                                     'hankittavan_ammat_tutkinnon_osan_osaamisen_hankkimistavat',
+                                     'hankittavat_ammat_tutkinnon_osat',
+                                     'hankittavat_yhteiset_tutkinnon_osat',
+                                     'osaamisen_hankkimistavat',
+                                     'sessions',
+                                     'yhteisen_tutkinnon_osan_osa_alueen_naytot',
+                                     'yhteisen_tutkinnon_osan_osa_alueen_osaamisen_hankkimistavat',
+                                     'yhteisen_tutkinnon_osan_osa_alueet')
+            LOOP
+                EXECUTE format('CREATE TABLE %I.%I AS SELECT * FROM public.%I', target_schema, tbl.table_name, tbl.table_name);
```

Comment:
Mä luulen et saat noi constraintit tässä samalla jos teet ensin jonku 
```suggestion
                EXECUTE format('CREATE TABLE %I.%I LIKE public.%l INCLUDING ALL', target_schema, tbl.table_name, tbl.table_name);
                EXECUTE format('INSERT INTO %I.%I SELECT * FROM public.%I', target_schema, tbl.table_name, tbl.table_name);
```